cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -467,6 +496,20 @@ func isDeviceStatusUpdated(oldTwin *v1alpha2.DeviceStatus, newTwin *v1alpha2.Dev
 	return !reflect.DeepEqual(oldTwin, newTwin)
 }
 
+// isDesiredTwinUpdated checks if desired twin is updated
+func isDesiredTwinUpdated(oldTwinDesired *v1alpha2.DeviceStatus, newTwin *v1alpha2.DeviceStatus) bool {
```

Comment:
`oldTwinDesired` name to `oldTwin` looks better

Reply:
Great suggestion! Make it later.